src/main/java/org/jabref/gui/importer/ImportEntriesViewModel.java

```diff
@@ -119,10 +119,10 @@ public void importEntries(List<BibEntry> entriesToImport, boolean shouldDownload
                                                      .anyMatch(this::hasDuplicate)).onSuccess(duplicateFound -> {
                 if (duplicateFound) {
                     boolean continueImport = dialogService.showConfirmationDialogWithOptOutAndWait(Localization.lang("Duplicates found"),
-                            Localization.lang("There are possible duplicates (marked with an icon) that haven't been resolved. Continue?"),
-                            Localization.lang("Continue with import"),
+                            Localization.lang("There are possible duplicates that haven't been resolved. Continue?"),
+                            Localization.lang("Auto merge"),
```

Comment:
Auto merge is probably not the right name for the button, since the new entry is just imported next to the old one if I remember correctly.

Reply:
Re-Import would be a better name?

Reply:
It's difficult, because its not a duplicate for sure, but JabRef only thinks, it is a duplicate. It is only a guess.

The user has to choice to think if they really want to import that stuff or not. No further information, how many entries are duplicates, which ones are duplicates, ... The dialog as is is not that helpful.

In other words: JabRef does not do any auto merge at all. It is also not a re-import, because the duplicates might not be "real" duplicates (letter-by-letter).

The decision the user has to take is: "Continue processing and all entries indepdenent of them being new ones or duplicates" or "Cancel import and leave the database as is"

These texts are too long for a button. Thus, we thought that "Continue with import" is a good word.

WDYT?

Reply:
Based on the current behavior of the app I believe is better to leave the previous message ("Continue with import"), I feel that is more descriptive of what is going to happen.